From 8f8c5397dcf1f163add78ef67c702a507a2057c7 Mon Sep 17 00:00:00 2001
From: drewromes <92449491+drewromes@users.noreply.github.com>
Date: Tue, 27 Oct 2026 23:58:38 -0400
Subject: [PATCH] Extra 1.0 units of insulin at insertion

Deliver an extra 1.0 units of insulin on insertion after 0.5 units of cannula priming
---
 OmniBLE/OmnipodCommon/Pod.swift | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/OmniBLE/OmniBLE/OmnipodCommon/Pod.swift b/OmniBLE/OmniBLE/OmnipodCommon/Pod.swift
index 75db1abc..90fbea72 100644
--- a/OmniBLE/OmniBLE/OmnipodCommon/Pod.swift
+++ b/OmniBLE/OmniBLE/OmnipodCommon/Pod.swift
@@ -82,7 +82,7 @@ public struct Pod {
     // Checked to verify it agrees with value returned by pod during the pairing process.
     public static let cannulaInsertionUnits = 0.5
 
-    public static let cannulaInsertionUnitsExtra = 0.0 // edit to add a fixed additional amount of insulin during cannula insertion
+    public static let cannulaInsertionUnitsExtra = 1.0 // edit to add a fixed additional amount of insulin during cannula insertion
 
     // Default and limits for expiration reminder alerts
     public static let defaultExpirationReminderOffset = TimeInterval(hours: 2)
